ckan/requirements.in

```diff
@@ -6,6 +6,9 @@ ckanext-dcat @ git+https://github.com/ckan/ckanext-dcat@master
 -e git+https://github.com/gsa/ckanext-spatial.git@datagov#egg=ckanext-spatial
 # git+https://github.com/keitaroinc/ckanext-saml2auth.git@main#egg=ckanext-saml2auth
 git+https://github.com/gsa/ckanext-saml2auth.git@main#egg=ckanext-saml2auth
+-e git+https://github.com/gsa/ckanext-qa.git@datagov#egg=ckanext-qa
+-e git+https://github.com/ckan/ckanext-archiver.git@master#egg=ckanext-archiver
+-e git+https://github.com/ckan/ckanext-report.git@master#egg=ckanext-report
```

Comment:
Just to make sure, do these *need* to be installed in editable mode? Did we try without? Either way, could we make a comment (either not tested, or with a note about errors when installing without `-e`?

Reply:
If it's not in editable mode, the custom templates are missing and can't be accessed.  So yeah, we need it in editable mode.  Also, the cli commands don't work if not editable.

(I did try it without editable initially)